src/wp-includes/media.php

```diff
@@ -5614,3 +5847,83 @@ function wp_increase_content_media_count( $amount = 1 ) {
 
 	return $content_media_count;
 }
+
+/**
+ * Determines whether to add `fetchpriority='high'` to loading attributes.
+ *
+ * @since 6.3.0
+ * @access private
```

Comment:
Why private access for this function?

Reply:
Just like `wp_increase_content_media_count` this function will only be ever used within the scope of `wp_get_loading_optimization_attributes` so `access private`